File: postgraduate/src/main/java/com/postgraduate/service/SelCourseService.java
package com.postgraduate.service;

import java.util.List;


import com.postgraduate.pojo.TbSelCourse;


public interface SelCourseService {
	/**
	 * 增加学生选课
	 * @return
	 */
	public void insert(int uid,int sid);
	/**
	 * 查询学生选课情况
	 * @return
	 */
	public List<TbSelCourse> selectAll(int id);
	/**
	 * 管理员查询学生选课情况
	 * @return
	 */
	public List<TbSelCourse> adminSelAll();

	/**
	 * 学生删除选课
	 * @return
	 */
	public void delete(int stuid,int lesteaid);
}

File: postgraduate/src/main/webapp/js/service/selCourseService.js
app.service("selCourseService",function($http){
	//增加
	this.insert=function(uid,sid){
		return $http.post('../SelCourse/insert.do?uid='+uid+'&sid='+sid);
	}
	//查询选课情况
	this.selectAll=function(id){
		return $http.post('../SelCourse/selectAll.do?id='+id);
	}	
	//管理员查询选课情况
	this.adminSelAll=function(){
		return $http.get('../SelCourse/adminSelAll.do');
	}
	this.selectOne=function(id){
		return $http.get('../Plesson/selectOne.do?id='+id);
	}
	//删除已选中课程
	this.studelete=function(stuid,lesteaid){
		return $http.post('../SelCourse/delete.do?stuid='+stuid+'&lesteaid='+lesteaid);
	}
});